langchain4j/src/main/java/dev/langchain4j/classification/LabelWithScore.java

```diff
@@ -0,0 +1,20 @@
+package dev.langchain4j.classification;
+
+public class LabelWithScore<E> {
```

Comment:
```suggestion
public class LabelWithScore<L> {
```

(`L` is for label type). Could you pelase rename `E` into `L` everywhere?

Reply:
I am also wondering if we need to rename the class since it might later become `LabelWithScoreWithXwithYWithZ...`

Reply:
> I am also wondering if we need to rename the class since it might later become `LabelWithScoreWithXwithYWithZ...`

LOL. Maybe rename it to `LabelResult`?

Reply:
Now I rename it to `ClassifyResult`, and rename the method to `classifyWithDetail`.